app/models/document_upload.rb

```diff
@@ -11,6 +11,7 @@ module Status
     QUEUED = 'queued'
     PROCESSING = 'processing'
     FINISHED = 'finished'
+    ERROR = 'error'
```

Comment:
Is this used anywhere?  Can we set the value on `document_upload.status`?

Reply:
Yes, you can see it on line #222 of the `lib/tasks/ingestor.rake` file